app/commands/v2/publish.rb

```diff
@@ -101,6 +102,18 @@ def build_live_attributes(draft_content_item)
 
         attributes
       end
+
+      def handle_path_change(live_content_item)
+        path_change = live_content_item.previous_changes[:base_path]
+        if path_change.present? && path_change[0].present?
+          draft_redirect = DraftContentItem.find_by(format: "redirect", base_path: path_change[0])
+          self.class.call(
+            content_id: draft_redirect.content_id,
+            locale: draft_redirect.locale,
+            update_type: "major"
```

Comment:
This is the `update_type` of the newly created `redirect` content item. It's different from the `update_type` of the content item that has moved. Given this is a brand new content item, it seems reasonable to use an `update_type` of `major`, although technically, it's not an update at all.